(Doc+) Link API doc to parent object - part1 (#111951)
elastic/elasticsearch: docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc

[[tutorial-migrate-data-stream-from-ilm-to-dsl]]
=== Tutorial: Migrate ILM managed data stream to data stream lifecycle 

> In this tutorial we'll look at migrating an existing data strea from <<index-lifecycle-management,Index Lifecycle Management ({ilm-init})>> to

Reviewer: ```suggestion
In this tutorial we'll look at migrating an existing data stream from <<index-lifecycle-management,Index Lifecycle Management ({ilm-init})>> to
```

> data stream lifecycle. The existing {ilm-init} managed backing indices will continue 

Author: ```suggestion
<<data-stream-lifecycle,data stream lifecycle>>. The existing {ilm-init} managed backing indices will continue 
```

[[tutorial-migrate-data-stream-from-ilm-to-dsl]]
=== Tutorial: Migrate ILM managed data stream to data stream lifecycle 

> In this tutorial we'll look at migrating an existing <<data-streams,data stream>> from <<index-lifecycle-management,Index Lifecycle Management ({ilm-init})>> to

Reviewer: ```suggestion
In this tutorial we'll look at migrating an existing data strea from <<index-lifecycle-management,Index Lifecycle Management ({ilm-init})>> to
```

> <<data-stream-lifecycle,data stream lifecycle>>. The existing {ilm-init} managed backing indices will continue 

Reviewer: ```suggestion
data stream lifecycle. The existing {ilm-init} managed backing indices will continue 
```
